easybuild/easyconfigs/n/ncurses/ncurses-6.0_gcc-5.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/ncurses/base/MKlib_gen.sh b/ncurses/base/MKlib_gen.sh
```

Comment:
@akesandgren please include a reference to where you got this patch, and a short description of what it fixes (any lines above the `---` line are comment, so just add some lines at the very top)